src/scenes/Languages/Edit/EditLanguageForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { UpdateLanguageInput } from '../../../api';
+import {
+  DialogForm,
+  DialogFormProps,
+} from '../../../components/Dialog/DialogForm';
+import { SubmitError, TextField } from '../../../components/form';
+import { LanguageQuery } from '../Overview/LanguageOverview.generated';
+
+export type EditLanguageFormProps = DialogFormProps<UpdateLanguageInput> & {
+  languageFormValues: UpdateLanguageInput;
```

Comment:
Just pass it as `initialValues` and then you don't need to define the extra prop

Reply:
I passed this in as initialValues to avoid the extra prop